modules/route_replies.cpp

```diff
@@ -369,7 +369,8 @@ class CRouteRepliesMod : public CModule
 		// The timer will be deleted after this by the event loop
 
 		if (!GetNV("silent_timeouts").ToBool()) {
-			PutModule("This module hit a timeout which is possibly a bug.");
+			PutModule("This module hit a timeout which is probably a connectivity issue.");
+			PutModule("However, if you can provide steps to reproduce this issue all the time, please do report a bug.");
```

Comment:
That's what "steps to reproduce" means. But "all the time" isn't even very English.